plugin/minimal/README.md

```diff
@@ -0,0 +1,31 @@
+# minimal-responses
+
+## Name
+
+*minimal-responses* - minimizes size of the DNS response message whenever possible.
+
+## Description
+
+The *minimal-responses* plugin tries to minimize the size of the response. Depending on the response type it removes resource records from the AUTHORITY and ADDITIONAL sections.
+
+
+## Syntax
+
+~~~ txt
+minimal-responses
+~~~
+
+## Examples
+
+Enable minimal-responses:
+
+~~~ corefile
+example.org {
+    whoami
+    minimal-responses
+}
+~~~
+
+## See Also
+
+[BIND 9 Configuration Reference](https://bind9.readthedocs.io/en/latest/reference.html#boolean-options)
```

Comment:
still bit hard to find the actual docs for minimal responses there. Is there a more direct link? Also interesting to see the amount of options there...

Reply:
I tried to find other links there was one but it wasn't directly from bind : https://www.zytrax.com/books/dns/ch7/queries.html#minimal-responses